src/Controller/Admin/AbstractBaseCrudController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\EditableEntityInterface;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Action;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Actions;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+
+abstract class AbstractBaseCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return self::class;
+    }
+
+    public function configureActions(Actions $actions): Actions
+    {
+        return parent::configureActions($actions)
+            ->update(Crud::PAGE_INDEX, Action::EDIT, static function (Action $action) {
+                return $action->displayIf(static function (object $entity) {
+                    $interfaces = class_implements($entity);
+
+                    return !isset($interfaces[EditableEntityInterface::class]) || $entity->isEditable();
```

Comment:
Why don't you use

```suggestion
                    return !($entity instanceof EditableEntityInterface) || $entity->isEditable();
```
?

Reply:
Because I like to take the long road ... and the PHP docs says:

> determine whether a PHP variable is an instantiated **object** of a certain **class**...

Not implements interface... but I see if you scroll down and down and down in the documentation... they use it on an interface as example... way down